### DISCUSSION TITLE
Stop trying to make a tool for work into Facebook For Code

### DISCUSSION THREAD

**Discussion by pikhq** [community/community#66203] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This is not a social media platform. No, honestly. It is not. This is not a toy I use to waste time, this is a tool I use in my work. Professionally. For pay. I would like to know about the things that happen in the repositories that I am an active developer in. What's next, social features in Word? In Excel? In Powerpoint? Think about the target audience before spending hours developing features that fundamentally do not make sense.

**Comment by karwa** [comment]
This isn't about the target audience; it's about product positioning. 

Microsoft want it to be a certain thing, and that's the driving force. It's not "how do we improve GitHub?"; it's "how do we turn GitHub in to X over 5 years?"
